### PR TITLE
Bug 1801064: Deprecate Pipelines in new-app/build

### PR DESCRIPTION
**Pull request by adambkaplan**
Add a warning to `oc new-app` and `oc new-build` if a Jenkins pipeline strategy build was created.

### CODE REVIEW

File: pkg/cli/newapp/newapp.go [hunk 1]
```diff
@@ -457,6 +457,9 @@ func (o *AppOptions) RunNewApp() error {
 				installing = append(installing, t)
 			}
 		case *buildv1.BuildConfig:
+			if t.Spec.Strategy.Type == buildv1.JenkinsPipelineBuildStrategyType {
+				fmt.Fprintf(out, "%sNOTICE: the JenkinsPipeline strategy is deprecated; consider using Jenkinsfiles directly on Jenkins or OpenShift Pipelines.\n", indent)
```
Comment (soltysh) [inline pkg/cli/newapp/newapp.go]: Log to `o.Action.ErrOut` and make the message similar to https://github.com/openshift/oc/blob/master/pkg/cli/rollout/latest.go#L114

File: pkg/cli/newbuild/newbuild.go [hunk 2]
```diff
@@ -228,6 +228,9 @@ func (o *BuildOptions) RunNewBuild() error {
 	for _, item := range result.List.Items {
 		switch t := item.(type) {
 		case *buildv1.BuildConfig:
+			if t.Spec.Strategy.Type == buildv1.JenkinsPipelineBuildStrategyType {
+				fmt.Fprintf(out, "%sNOTICE: the JenkinsPipeline strategy is deprecated; consider using Jenkinsfiles directly on Jenkins or OpenShift Pipelines.\n", indent)
```
Comment (soltysh) [inline pkg/cli/newbuild/newbuild.go]: Ditto.

File: pkg/cli/newapp/newapp.go (hunk 1 above)
Comment (adambkaplan) [reply pkg/cli/newapp/newapp.go]: @soltysh done in latest push.

**Pull request merged**